test/test_plugin_emby.py

```diff
@@ -70,6 +71,11 @@
     }),
 )
 
+PY311 = sys.version_info[0:2] >= (3, 11)
+EMBY_CLASS = 'apprise.plugins.NotifyEmby'
+if PY311:
+    EMBY_CLASS = 'apprise.plugins.NotifyEmby.NotifyEmby'
```

Comment:
Now that is weird, no? Either Python, or `unittest`, seems to get confused about modules vs. classes? Actually, I think Python 3.11 is correct, because the address of the class indeed would be `apprise.plugins.NotifyEmby.NotifyEmby`, right?

But wait, there is `assert isinstance(obj, plugins.NotifyEmby)`, which signals that `apprise.plugins.NotifyEmby` is actually a class!? Does Apprise apply any magic intending to "overlay" the `apprise.plugins.NotifyEmby` module with the `apprise.plugins.NotifyEmby.NotifyEmby` class somehow?

Reply:
Not sure what's going on... seems really weird that it's giving you so much difficulties.

if you open up `apprise/plugins/__init__.py` you'll see the magic that allows all of the librarys to dynamically load.  It strips off the need to do `apprise.plugins.NotifyPlugin.NotifyPlugin`.  It's here that makes it so to add/remove a service to apprise simply involves droping a new `.py` file in place in the `/plugins/` path.

Reply:
Hi Chris, thanks for the hint. I have been able to improve the magic plugin loader with 5e5ae7f.

Reply:
... not there yet. Will continue later.

Reply:
Wow what should be easy really turned out to be a pain.  I wonder what the justification is to change the library references as much as they have between these versions?  Is it possible it's a bug with Python that is overlooked in the new dev branch?  I can't seem to find any white-papers describing why/where the module inclusion and it's name-spacing was changed upsteam.

Reply:
Hi Chris. I started writing a response here, but it got a bit lengthy, so I diverted it to #685. Cheers, Andreas.

Reply:
> This looks fine to me; was there anything else you wanted to add, or is this a good for merging?

I just found a minimal solution to get this in, which aligns with, and can later on be expanded into a more general #685. I may be able to submit an update later this night.

Reply:
I think it will be good to go now. Some of the adjustments may look weird and unrelated, but C'est la vie ;]. I am happy to have been able to make it work without another larger detour.